src/app/shared/view-issue/new-team-response/new-team-response.component.html

```diff
@@ -5,7 +5,7 @@
     </div>
 
     <div>
-      <div class="container">
+      <div *ngIf="(issueService.getDuplicateIssuesFor(this.issue) | async).length === 0" class="container">
```

Comment:
```suggestion
      <div *ngIf="(issueService.getDuplicateIssuesFor(this.issue) | async).length === 0; else chooseDuplicates" class="container">
```

Let's use the ngtemplate if-else to relate the 2 html components together

Reply:
ok! thanks for this